README.md

```diff
@@ -1,13 +1,229 @@
 # CHINESE MENU GENERATOR-BACKEND
 
-## ERD
+#### Team Building API Exercise Using NodeJS/ExpressJS
+
+<hr style="border: solid 1px"/>
+
+## Table of Content
+
+- [Summary](#summary)
+- [Authors](#authors)
+- [Goals](#goals)
+  - [Project First Itearation Goals](#project-first-iteration-goals)
+  - [Extension Goals for Future Iterations](#extension-goals-for-future-iterations)
+- [Stack](#stack)
+- [Install Locally](#install-locally)
+- [Schema](#schema)
+- [Swagger](#swagger)
+- [Endpoints](#endpoints)
+  - [Postman](#postman)
+  - [User Endpoints](#user-endpoints)
+  - [Menu Endpoints](#menu-endpoints)
+  - [Recipe Endpoints](#recipe-endpoints)
+
+<hr style="border: solid 1px"/>
+
+## Summary
+
+The main goal of this exercise was to build an API that in NodeJS and ExpressJS that uses TypeScript including tests - Jest - and documentation - swagger.
+
+As a secondary goal was to learn how to work together so we can successfully complete larger projects in the future. To achieve this we drew from each other's work and life experience and we were able to establish patterns that worked well for us and facilitated fluid communication and standards clarity.
```

Comment:
Should we remove the "As"?  Just say " The secondary goal..."? 
A little update for the secondary goal: 

The secondary goal of our project was to enhance our ability to work together effectively, enabling us to successfully complete larger projects in the future. To accomplish this, we leveraged our collective work and life experiences, allowing us to incorporate valuable insights from each team member. As a result, we established productive patterns that fostered seamless communication and clarity in standards.

Reply:
I like it. Changing and pushing